docs/source/cluster_management/sweep/targeted-sweep.rst

```diff
@@ -91,4 +86,4 @@ The safe way to change the sweep strategy from THOROUGH to CONSERVATIVE:
 
 The safe way to change sweep strategy from CONSERVATIVE to THOROUGH:
    1. Shut down all the nodes.
-   2. Start AtlasDB with the new table metadata.
```

Comment:
I don't know why this line shows up as having changed.

Reply:
Neither do I (even checking for tabs against spaces)!